src/Uno.Toolkit.RuntimeTests/Tests/TabBarTests.cs

```diff
@@ -388,5 +391,40 @@ public async Task Verify_Indicator_Placement(IndicatorPlacement placement)
 				}
 			}
 		}
+
+		[TestMethod]
+		public async Task Verify_SelectedIndex_Not_Set_Unnecessarily()
+		{
+			var item1 = new TabBarItem();
+			var item2 = new TabBarItem();
+
+			var SUT = new TabBar();
+			SUT.Items.Add(item1);
+			SUT.Items.Add(item2);
+			SUT.DataContext = new MyViewModel();
+			SUT.SetBinding(TabBar.SelectedIndexProperty, new Binding() { Mode = BindingMode.OneWay, Path = new PropertyPath("P") });
+
+			await UnitTestUIContentHelperEx.SetContentAndWait(SUT);
+			Assert.IsNotNull(SUT.GetBindingExpression(TabBar.SelectedIndexProperty));
+			SUT.SelectedIndex = 0;
+			// Possible Uno bug? Explicit setting of SelectedIndex should have cleared the binding.
```

Comment:
Should we log an issue for this? Does it work on Windows?

Reply:
setting value (almost?) never clears the binding; in fact, that how you get value propagated through the binding(s)
if you want to clear the bindings, call `SUT.ClearValue(TabBar.SelectedIndexProperty);`

Reply:
@Xiaoy312 This is what I noticed on WinUI. It gets cleared for OneWay, but not TwoWay.

Reply:
My test on Windows was having a `<TextBox x:Name="tb" Text="{x:Bind Something, Mode=OneWay}" />`, then get the binding expression and find it's non-null, then set `tb.Text = something`, then get the binding expression again and find it's null